css-three-column-layout/styles.css

```diff
@@ -0,0 +1,50 @@
+.container {
+  width: 100%;
+  height: 600px;
+}
+
+body {
+  width: 80%;
```

Comment:
You shouldn;t set width on the body element, you should switch this to the `container` class